### ISSUE TITLE
Game crashes and reaches unrecoverable state often

### ISSUE THREAD

**Issue by gbom** [FredrIQ/fiqhack#43]
I have 3 games in an unrecoverable state at eu.hardfought.org. The character names are Udome, Kiemg, and Amper. Attempting to load the games always crashes FIQHack and takes us back to the hardfought game selection screen.

How they happened: two of the there times, I was looking at the contents from some container, IIRC it was a magic box at Minetown or Sokoban. I forgot how the other one happened; it was a long time ago.

For an example of where the game **always** crashes but recovers: getting escorted out of a vault by a guard through the walls that are supposed to close up as soon as we exit them. The game crashes one or two cells before we can exit the temporary path.

**Comment by FredrIQ** [comment]
Hi.
Just to be clear, the magic chest crash was for an old issue a long time ago? Because in that case, it's likely a result from a now-fixed bug, and a simple manual rollback from the server admin should address it.

I'll try to investigate the vault guard problem.

**Comment by gbom** [comment]
No, the last two magic chest crashes happened just yesterday and last week. I can't load any of the three games.

**Comment by gbom** [comment]
I may have misunderstood your last question. Unrecoverable crashes were very rare a few years ago. I think I have asked the server admins to delete a couple of my unrecoverable game files just once. And because the crashes happened such a long time ago, I don't recall what actions led to them. All I recall was that attempting to re-load the savefiles would crash FIQHack too. At the moment, I have three such savefiles at eu.hardfought.org, two of them generated in the last few days.
